daphne_worker/src/dap.rs

```diff
@@ -64,32 +65,117 @@ pub(crate) fn dap_response_to_worker(resp: DapResponse) -> Result<Response> {
     Ok(worker_resp)
 }
 
+#[async_trait(?Send)]
 impl<D> HpkeDecrypter for DaphneWorkerConfig<D> {
-    fn get_hpke_config_for(&self, _task_id: &Id) -> Option<&HpkeConfig> {
-        if self.hpke_receiver_config_list.is_empty() {
-            return None;
-        }
+    async fn get_hpke_config_for(
```

Comment:
Please add a test to `e2e.sh` that confirms the receiver config is written to KV. It's sufficient to confirm that each call to `GET /hpke_config` returns the same config.

Reply:
The requested test has been added :)